src/mixxx.cpp

```diff
@@ -261,6 +261,7 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
                              m_pRecordingManager);
     m_pPlayerManager->bindToLibrary(m_pLibrary);
 
+    m_pLibrary->scan();
```

Comment:
Yes I checked recently suppose I have the directory, the music directory, if I deleted a song in the directory then I do rescan then the deleted file goes to the Missing Tracks,